nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestListenHTTP.java

```diff
@@ -89,7 +90,7 @@ public void testPOSTRequestsReceivedReturnCodeWithoutEL() throws Exception {
         runner.setProperty(ListenHTTP.BASE_PATH, HTTP_BASE_PATH);
         runner.setProperty(ListenHTTP.RETURN_CODE, Integer.toString(204));
 
-        testPOSTRequestsReceived(204);
+        testPOSTRequestsReceived(HttpServletResponse.SC_OK);
```

Comment:
should this have been HttpServletResponse. SC_NO_CONTENT?  The line removed had a code of 204 but this is now passing in a 200.